packages/common/package.json

```diff
@@ -27,10 +27,14 @@
     "clean": "rm -rf ./lib"
   },
   "dependencies": {
-    "@ceramicnetwork/docid": "^0.5.1-rc.0",
+    "@ceramicnetwork/docid": "^0.5.1-rc.3",
```

Comment:
this file and the change to `logger/package.json` are the only real substantive changes, the rest is all version bumps from doing RC releases to test the webpack build